dxr/plugins/rust/tests/test_cargo/test_cargo.py

```diff
@@ -0,0 +1,17 @@
+from dxr.testing import DxrInstanceTestCase
+import os
+from os.path import dirname, join
+import subprocess
+import sys
+
+class CargoTests(DxrInstanceTestCase):
+    @classmethod
+    def setup_class(cls):
+        os.environ["RUSTC"] = "./dxr-rustc.sh"
```

Comment:
(Also per IRC) it sounds like RUSTC_ARGS is coming to Cargo in the near-ish future. Therefore I'll leave the painful script approach for now (rather than shipping the script with DXR) with the expectation that I will move to the env var approach asap.